knexfile.js

```diff
@@ -0,0 +1,31 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'pg',
+    connection: 'postgres://localhost/quantified_self',
+    migrations: {
+      directory: './db/migrations'
+    },
+    seeds: {
+      directory: './db/seeds/dev'
+    },
+    useNullAsDefault: true
+
+  },
+
+  production: {
```

Comment:
The setup for production environment is not totally set up I'm missing the name of the database in Heroku, but heroku won't create one until we push from master